java/java.nativeimage.debugger/src/org/netbeans/modules/java/nativeimage/debugger/breakpoints/JPDABreakpointsHandler.java

```diff
@@ -154,6 +159,25 @@ private Breakpoint added(JPDABreakpoint b) {
         return null;
     }
 
+    private String verifyFilePath(String filePath, String relativePath) {
+        if (relativePath != null) {
+            List<SourceInfo> sources = debugger.listSources(SOURCES_FOLDER + File.separator + relativePath);
```

Comment:
`listSources` is supposed to take RegExp - what will it do on Windows? `\` is an escape character...

Reply:
Good point.
After all I've abandoned matching file paths in the sources listing as we can not have reliable source list for lazily loaded shared libraries.